### PR TITLE
fix(theme): tabs with anchor due to react-bootstrap

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
`react-bootstrap` uses anchors for tabs and we only customize style for `<button>`

**What is the new behavior?**
Our theme customizes anchors for `tabs` Bootstrap components.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:
Depends on #129 to be tested on Surge.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

File: packages/theme/src/theme/_navs.scss [hunk 1]
```diff
@@ -113,6 +97,39 @@
...
+					&:hover {
+						background: transparent;
+						border: none;
+					}
+				}
+			}
+		}
+
+		+ .tab-content {
+			padding-top: $padding-larger;
```
Comment (talend-sromain) [inline packages/theme/src/theme/_navs.scss]: It isn't too much padded with this padding-top ?

Comment (frassinier) [inline packages/theme/src/theme/_navs.scss]: No it's because of Bootswatch example embedded style. Let me fix that. :ok_hand:

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://130.talend.surge.sh)

**Pull request merged**